### ISSUE TITLE
"target=_blank" not respected in Styled Pages

### ISSUE THREAD

**Issue by chucklapointe** [narcitymedia/lilium-cms#347]
**Describe the bug**

It seems like the target="_blank" parameter attached to links is not being respected when set in Styled Pages.

**To Reproduce**
1. Go to Styled page in Lilium: https://www.narcity.com/lilium/styledpages/edit/5caa673c7f11f7738ea3b9e6
2. Scroll down and check for "Toronto", you will see it has a New Window attribute
3. Visit page [here](https://www.narcity.com/villes) and click on Toronto
4. The link does not open in a new tab, which creates a problem

**Expected behavior**

If there are "New Window" parameters set in TinyMCE or in the source code, they should be respected.

**Section**

Styled pages

**Comment by rykdesjardins** [comment]
Yeah that's right, the moment a link references a page from the website, it will load the page inside the app instead of doing a full refresh.

There might be a way to instruct Polemos to forcefully load the URL with a full navigation.

**Comment by rykdesjardins** [comment]
@chucklapointe I found a pretty interesting solution that will offer a quick fix. Since the idea of a front-end rendered site is to avoid having to do a second page load, wanting to do so is an edge case. 

I just implemented something in our Polemos virtual DOM to allow a target blank for same-origin links. 

Simply prepend a "=" symbol before any path you want a hard refresh for. Make sure not to include the full URL, and just the path like so : `/local/ca/on/toronto`. In order to cause a hard refresh, simply use : `=/local/ca/on/toronto`.

Let me know if you're happy with this solution. If so, I'll create a PR tonight!

**Comment by chucklapointe** [comment]
Personally, this solution works for me!

However, I am convinced other people using Lilium (such as The961.com) will
expect some pages to load in a new tab when they specify "Open in new tab"
or "Open in new window" in TinyMCE. We might want to remove that option if
it's not possible for internal links (or at least give a disclaimer
somewhere in that link popup for internal links).

**Comment by rykdesjardins** [comment]
100%! Since it's a theme feature and not directly included in Lilium, this can be changed and the future and parameterised, too.
